tools/nodejs/configure

```diff
@@ -5,10 +5,4 @@ set -x
 
```

Comment:
We can probably just kill this file, right?

Reply:
I kept it around for backwards compatibility purposes - but I suppose we could replace it with the `configure.py` as well